### PR TITLE
Update Hub function that updates existing registration

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
- update_registration should be renamed to create_change.
- object parameters should be passed in and post'ed as is

should look like:
```
 create_registration: function(reg_info) {
        var endpoint = 'registration/';
        var url = this.base_url + endpoint;

        return this.http_api.post(url, reg_info)
            .then(this.log_service_call('POST', url, reg_info, null))
            .then(function(response) {
                return response.data;
            });
    },

    /**:Hub.update_registration(reg_update_data)
    Updates existing registration

    :param object reg_update_data:
      Gets passed through directly to service. Consult service docs/protocol.
    */
    create_change: function(reg_update_data) {
        var endpoint = 'change/';
        var url = this.base_url + endpoint;

        return this.http_api.post(url, reg_update_data)
            .then(this.log_service_call('POST', url, reg_update_data, null))
            .then(function(response) {
                return response.data;
            });
    },
```

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr please review.

File: test/test_utils.js [hunk 1]
```diff
@@ -1016,12 +1016,12 @@ describe("Testing app- and service call functions", function() {
...
-                                "stage": "postbirth",
-                                "mother_id": "cb245673-aa41-4302-ac47-1234567890"
+                        return hub.create_change({
+                                "mother_id": "cb245673-aa41-4302-ac47-1234567890",
+                                "action": "change_stage"
                             })
-                            .then(function(registration) {
-                                assert.equal(registration.id, "reg_for_00000000002_uuid");
+                            .then(function(response) {
+
```
Comment (gsvr) [inline test/test_utils.js]: Please assert what is in response here.

**Comment by gsvr** [comment]
There may be an issue with the code I suggested - let's discuss on Tuesday

**Comment by gsvr** [comment]
As discussed, it looks like we need to put `{data: ... }` back in, sorry.

**Comment by NicolaasDieBaas** [comment]
@gsvr please review.  I put {data: ... } back in, did the assert you asked for and also sneaked in a rename of 'prefix' to 'url'

**Comment by gsvr** [comment]
:+1:

**Pull request merged**